homeassistant/components/here_travel_time/sensor.py

```diff
@@ -240,6 +240,17 @@ def __init__(
         self._attr_native_unit_of_measurement = TIME_MINUTES
         self._attr_name = name
 
+    async def async_added_to_hass(self) -> None:
+        """Handle when entity is added."""
+        await super().async_added_to_hass()
+        if self.hass.state != CoreState.running:
+            _LOGGER.info("Waiting for Home Assistant to fully start")
+            self.hass.bus.async_listen_once(
+                EVENT_HOMEASSISTANT_STARTED, self.async_update()
```

Comment:
Should this be just `self.async_update`?

```suggestion
                EVENT_HOMEASSISTANT_STARTED, self.async_update
```

I'm thinking it will call the method itself, and this matches how other instances look.